source/_integrations/home_connect.markdown

```diff
@@ -103,7 +103,50 @@ The integration configuration will ask for the *Client ID* and *Client Secret* c
+| `consumer_products_coffee_maker_option_bean_container` | yes | Defines the preferred bean container. |
+| `consumer_products_coffee_maker_option_flow_rate` | yes | Defines the water-coffee contact time. The duration extends to coffee intensity. |
+| `consumer_products_coffee_maker_option_multiple_beverages` | yes | Defines if double dispensing is enabled. |
+| `consumer_products_coffee_maker_option_coffee_milk_ratio` | yes | Defines the milk amount. |
+| `consumer_products_coffee_maker_option_hot_water_temperature` | yes | Defines the temperature suitable for the type of tea. |
+| `b_s_h_common_option_start_in_relative` | yes | Defines in how many time the program should start. |
+| `dishcare_dishwasher_option_intensiv_zone` | yes | Defines if the cleaning is done with higher spray pressure on the lower basket for very dirty pots and pans. |
+| `dishcare_dishwasher_option_brilliance_dry` | yes | Defines if the program sequence is optimized with special drying cycle ensures more shine on glasses and plastic items. |
+| `dishcare_dishwasher_option_vario_speed_plus` | yes | Defines if the program run time is reduced by up to 66% with the usual optimum cleaning and drying. |
+| `dishcare_dishwasher_option_silence_on_demand` | yes | Defines if the extra silent mode is activated for a selected period of time. |
+| `dishcare_dishwasher_option_half_load` | yes | Defines if economical cleaning is enabled for smaller loads which reduces energy and water consumption and also saves time. The utensils can be placed in the upper and lower baskets. |
+| `dishcare_dishwasher_option_extra_dry` | yes | Defines if improved drying for glasses and plasticware is enabled. |
+| `dishcare_dishwasher_option_hygiene_plus` | yes | Defines if the cleaning is done with increased temperatures which ensures maximum hygienic cleanliness for regular use. |
+| `dishcare_dishwasher_option_eco_dry` | yes | Defines if the door is opened automatically for extra energy efficient and effective drying. |
+| `dishcare_dishwasher_option_zeolite_dry` | yes | Defines if the program sequence is optimized with special drying cycle ensures improved drying for glasses, plates and plasticware. |
+| `laundry_care_dryer_option_drying_target` | yes | Describes the drying target for a dryer program. |
+| `cooking_hood_option_venting_level` | yes | Defines the required fan setting. |
+| `cooking_hood_option_intensive_level` | yes | Defines the intensive setting. |
+| `cooking_oven_option_setpoint_temperature` | yes | Defines the target cavity temperature, which will be hold by the oven. |
+| `b_s_h_common_option_duration` | yes | Defines the run-time of the program. Afterwards the appliance is stopped. |
+| `cooking_oven_option_fast_pre_heat` | yes | Defines if the cooking compartment is heated up quickly. Please note that the setpoint temperature has to be equal or higher than 100 °C or 212 °F otherwise the fast pre-heat option is not activated. |
```

Comment:
```suggestion
| `cooking_oven_option_fast_pre_heat` | yes | Defines if the cooking compartment is heated up quickly. Please note that the setpoint temperature has to be equal to or higher than 100 °C or 212 °F. Otherwise, the fast pre-heat option is not activated. |
```